doc/tomb.1

```diff
@@ -38,18 +38,18 @@ data gathered from a non-blocking source (/dev/urandom).
 .IP "forge"
 Creates a new \fIkey\fR and prompts the user for a \fIpassword\fR to
 protect its usage using symmetric encryption. This operation uses
-random data from a blocking source (/dev/random) and it may take long
-when run on a server with low entropy; to switch using a non-blocking
-source the \fI--use-urandom\fR flag can be used. The \fI-g\fR option
-switches on the use of a GPG key instead of a password (asymmetric
-encryption), then the \fI-r\fR option indicates the recipient key;
-more recipient GPG ids can be indicated (comma separated). The default
-cipher to protect the key is AES256, a custom one can be specified
-using the \fI-o\fR option, for a list of supported ciphers use
-\fI-v\fR. For additional protection against dictionary attacks on
-keys, the \fI--kdf\fR option can be used when forging a key, making
-sure that the \fItomb-kdb-pbkdf2\fR binaries in \fIextras/kdf\fR were
-compiled and installed on the system.
+random data from a non-blocking source (/dev/urandom) and it may take
+long only in some cases; to switch using a blocking source the
```

Comment:
Does it still takes a long time? My experience so far was, that only the usage of `/dev/random` could create those long times.